test/actions/katello/host/hypervisors_update_test.rb

```diff
@@ -8,7 +8,9 @@ class HypervisorsUpdateTest < ActiveSupport::TestCase
     include FactImporterIsolation
     allow_transactions_for_any_importer
 
-    before :each do
```

Comment:
Took the opportunity to refactor this from spec style to unit style testing so its easier to run individual tests + more consistent with most of the codebase